zmon_worker_monitor/builtins/plugins/time_.py

```diff
@@ -30,12 +33,20 @@ def create(self, factory_ctx):
 
 
 class TimeWrapper(object):
-    def __init__(self, spec='now', utc=False):
-        now = (datetime.utcnow() if utc else datetime.now())
+
+    EPOCH = datetime.fromtimestamp(0, pytz.UTC)
+
+    def __init__(self, spec='now', utc=False, tz_name=None):
+        if utc and tz_name:
+            raise ValueError('Ambiguous time zone. Don''t use "utc" and "tz_name" parameter at the same time.')
```

Comment:
please use `"""` or `'''` to quote strings with quotes (or simply use `Do not`)

Reply:
Good point.